src/test/e2e/reporter-config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "reporterEnabled": "spec, mochawesome",
+    "mochawesomeReporterOptions": {
+        "reportDir": "cypress/reports/mochawesome",
```

Comment:
I don't seem to be getting this mochawsome report when I run the tests. Should a report appear this in this path after each run?

Reply:
Weirdly I am also still not seeing screenshots on a failure

Reply:
![image](https://github.com/OWASP/wrongsecrets/assets/12913562/3cecd1f0-e9f8-4a0b-9299-2006badc4ba2)
This is my path after a failure